demo/build.gradle.kts

```diff
@@ -87,7 +87,8 @@ dependencies {
 
   implementation("com.google.android.fhir:data-capture:1.0.0")
   // {
-  //     exclude(group = Dependencies.androidFhirGroup, module = Dependencies.androidFhirEngineModule)
+  //     exclude(group = Dependencies.androidFhirGroup, module =
+  // Dependencies.androidFhirEngineModule)
```

Comment:
Why was removing this needed ?

Reply:
Anchita's change was just a spotlessApply. 
Are you asking about commenting that section? It was originally a workaround that Aditya recommended to get workflow to work with engine at HEAD which does not appear to be needed now. (You remember those caffeine errors?)

Reply:
Ah yes!